eva-accession-ws/src/main/java/uk/ac/ebi/eva/accession/ws/rest/ClusteredVariantsRestController.java

```diff
@@ -135,5 +143,66 @@ public ClusteredVariantsRestController(
                                 .map(wrapper -> new AccessionResponseDTO<>(wrapper, SubmittedVariant::new))
                                 .collect(Collectors.toList());
     }
+
+    @ApiOperation(value = "Find a clustered variant (RS) by the identifying fields", notes = "This endpoint returns "
+            + "the clustered variant (RS) represented by a given identifier. For a description of the response, see "
+            + "https://github.com/EBIvariation/eva-accession/wiki/Import-accessions-from-dbSNP#clustered-variant-refsnp"
+            + "-or-rs")
+    @GetMapping(produces = "application/json")
+    public ResponseEntity getByIdFields(
+            @RequestParam(name = "assemblyId") String assembly,
+            @RequestParam(name = "referenceName") String chromosome,
+            @RequestParam(name = "start") long start,
+            @RequestParam(name = "variantType") VariantType variantType) {
+        try {
+            AccessionResponseDTO<ClusteredVariant, IClusteredVariant, String, Long> clusteredVariant = beaconService
+                    .getClusteredVariantByIdFields(assembly, chromosome, start,variantType);
+            return clusteredVariant != null ? ResponseEntity.ok(clusteredVariant) : ResponseEntity.notFound().build();
+        } catch (Exception e) {
```

Comment:
do you need this? have you tried relying on the `EvaControllerAdvice`? it should return better error codes.